src/components/ReleaseService/ReleasePlan/ReleasePlanListView.tsx

```diff
@@ -25,17 +26,32 @@ import { ReleaseKind } from '../../../types';
 import { withPageAccessCheck } from '../../PageAccess/withPageAccessCheck';
 import { ReleaseServiceEmptyState } from '../ReleaseServiceEmptyState';
 import ReleasePlanListHeader from './ReleasePlanListHeader';
-import ReleasePlanListRow from './ReleasePlanListRow';
+import ReleasePlanListRow, { ReleasePlanWithApplicationData } from './ReleasePlanListRow';
 
 const ReleasePlanListView: React.FC<React.PropsWithChildren<unknown>> = () => {
   const namespace = useNamespace();
+  const [applications, appLoaded] = useApplications(namespace);
```

Comment:
use this in the ReleasePlanRow component.

Reply:
If I call this in `ListRow` level, it will be called `n` number of times whereas now it is being called only once and the data is being appended to the Row data